readme.md

```diff
@@ -1,26 +1,36 @@
 ##GitFun
 
-GitFun is an interactive git workshop following the style of [githug](https://github.com/Gazler/githug). gitfun is geared towards understanding the git workflow in a team, and how to deal with common problems and errors.
+GitFun is an interactive git workshop following the style of [githug](https://github.com/Gazler/githug). GitFun is geared towards understanding the git workflow in a team, and how to deal with common problems and errors.
 
-### To run
+### Setup
 
-Note: This workflow is subject to change as the project grows
+Note: This workflow is subject to change as the project grows.
 
-```
-$ git clone https://github.com/kmacpher/gitfun
-$ cd gitfun
-$ npm link or $ npm install
-Then from any directory, you can run gitfun as a command line tool
-$ gitfun
+```sh
+git clone https://github.com/kmacpher/gitfun
+cd gitfun
+npm link
```

Comment:
It would have to be `npm install -g`, not just `npm install`. I think it's simpler to just tell them to `npm link`. In the future it would make sense to publish this as an npm package, no? Then they could simply do `npm install -g gitfun` without any need for repo cloning and linking.